packages/e2e-test-utils-playwright/src/editor/toggle-fixed-toolbar.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import type { Editor } from './index';
+
+/**
+ * Toggles the fixed toolbar option.
+ *
+ * @param this
+ * @param isFixed Boolean value true/false for on/off.
+ */
+export async function toggleFixedToolbar( this: Editor, isFixed: boolean ) {
```

Comment:
This is taken from https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-tests/specs/editor/various/links.test.js#L275-L284. If the plan is to move the older e2e tests into playwright, then this will be useful.

Reply:
Maybe we should rename it to `setIsFixedToolbar` because `toggle` seems a bit weird with the `isFixed` parameter 🤔 .